trieste/models/keras/models.py

```diff
@@ -369,8 +416,13 @@ def __init__(
         )
 
         self.num_passes = num_passes
+        self.learning_rate = learning_rate
```

Comment:
Mmh, this is a valid concern. How should we go about this, check if a non-default rate is set in the optimizer, otherwise set it in the model? How did your approach work around this?

Reply:
You could just read the learning rate from the optimizer regardless if it is default or not and save that rate.